packages/block-library/src/query-title/edit.js

```diff
@@ -109,18 +132,33 @@ export default function QueryTitleEdit( {
 		titleElement = (
 			<>
 				<InspectorControls>
-					<PanelBody title={ __( 'Settings' ) }>
-						<ToggleControl
-							__nextHasNoMarginBottom
+					<ToolsPanel
+						label={ __( 'Settings' ) }
+						resetAll={ () =>
+							setAttributes( {
+								showSearchTerm: false,
+							} )
+						}
+					>
+						<ToolsPanelItem
+							hasValue={ () => showSearchTerm !== undefined }
```

Comment:
```suggestion
							hasValue={ () => showSearchTerm !== undefined }
```

It should be considered the opposite of a default value, i.e. it "has a value" when it is `false`.